diff --git a/quiche/BUILD.bazel b/quiche/BUILD.bazel
index 39d0de2..efb58fb 100644
--- a/quiche/BUILD.bazel
+++ b/quiche/BUILD.bazel
@@ -34,7 +34,7 @@ load("//build:test.bzl", "test_suite_from_source_list")
 licenses(["notice"])
 
 package(
-    default_visibility = ["//visibility:private"],
+    default_visibility = ["//visibility:public"],
     features = [
         "parse_headers",
         "layering_check",
